File: ueueo-framework/ueueo-domain-driven-desgin/src/main/java/com/ueueo/ddd/domain/entities/events/EntityEventReport.cs
using System.Collections.Generic;

namespace Volo.Abp.Domain.Entities.Events;

public class EntityEventReport
{
    public List<DomainEventEntry> DomainEvents;//  { get; }

    public List<DomainEventEntry> DistributedEvents;//  { get; }

    public EntityEventReport()
    {
        DomainEvents = new List<DomainEventEntry>();
        DistributedEvents = new List<DomainEventEntry>();
    }

    @Override public String toString()
    {
        return $"[{nameof(EntityEventReport)}] DomainEvents: {DomainEvents.Count}, DistributedEvents: {DistributedEvents.Count}";
    }
}

File: ueueo-frameworks/ueueo-ddd-domain/src/main/java/com/ueueo/ddd/domain/entities/events/EntityEventReport.java
package com.ueueo.ddd.domain.entities.events;

import lombok.Getter;

import java.util.ArrayList;
import java.util.List;

@Getter
public class EntityEventReport {
    private List<DomainEventEntry> domainEvents;

    private List<DomainEventEntry> distributedEvents;

    public EntityEventReport() {
        domainEvents = new ArrayList<>();
        distributedEvents = new ArrayList<>();
    }

    @Override
    public String toString() {
        return String.format("[EntityEventReport] DomainEvents: %s, DistributedEvents: %s", domainEvents.size(), distributedEvents.size());
    }
}
